a3_procurement/doc_events/sales_invoice.py

```
@@ -21,37 +59,37 @@ def patch_india_compliance_tax(doc, method=None):
 # ------------------------------------------------------------------------------
 
 def patch_ic_validate_item_wise_tax_detail():
-    try:
-        import india_compliance.gst_india.overrides.transaction as ic_trx
-    except Exception:
-        return
+	try:
+		import india_compliance.gst_india.overrides.transaction as ic_trx
+	except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)